open-api/rest-catalog-open-api.yaml

```diff
@@ -673,16 +726,13 @@ paths:
         404:
           description:
             Not Found
-            - NoSuchPlanIdException, the plan-id does not exist
```

Comment:
@rdblue Just curious why this was removed? I think if a user submits an invalid id then the service should throw some `404` for this right?

Reply:
This was removed because cancellation should never fail, it should just be a noop.

There is no value in throwing an exception when this is a hint from the client that the client doesn't care about the plan-id anymore and the server should release resources. If the server doesn't know what plan-id the client passes, it doesn't need to cause any issue on the client side.